data/kubemacpool/kubemacpool.yaml

```diff
@@ -143,6 +143,8 @@ spec:
         app: kubemacpool
         control-plane: cert-manager
         controller-tools.k8s.io: "1.0"
+      annotations:
+        openshift.io/required-scc: "restricted-v2"
```

Comment:
aint it should be with
`{{ if .IsOpenshift }}` or `{{ if .EnableSCC }}` ?

do we need to create this scc ? or it is one that is already created ?
(for example for bridge-marker we create the scc by CNAO upon needs)

Reply:
The script that templates this doesn't know any of those variables.

And kubemacpool - for whatever reason - already always has OpenShift related data set: https://github.com/kubevirt/cluster-network-addons-operator/blob/d5143952e14f7094a1a674ff7d6950aa4e738f37/data/kubemacpool/kubemacpool.yaml#L359

Reply:
if needed we can add to the script, i can show you if you want where

Reply:
why is this an issue now, when it wasn't before ?

Reply:
```
    - key: openshift.io/run-level
      operator: NotIn
      values:
      - "0"
      - "1"
```
for me this seems to have less effect then creating scc
wdty?
i am totally not into being more restrictive by this PR, but imho this might not be the same case here

Reply:
I'm not creating an SCC here, I'm saying that this pod can only use this SCC. 

On a platform that doesn't use SCCs - e.g. vanilla kubernetes - this is just one more annotation. There's no handler for it.

Reply:
btw, if KMP annotation can always be in the manifest, and no scc need to be created for this specific name
(iiuc it is a system one), then it should be better to not annotate it via CNAO, but add it directly on KMP repo please

Reply:
before making me spend a lot of time on this, can I get agreement from the maintainers of those repos that they're OK with it ? 

... like, if they're not, they'll have to do this themselves in 4.19 scope ... 

@phoracek / @RamLavi / @oshoval 
who maintains the linux-bridge thing ?

Reply:
of course you do, thanks
linux-bridge is part of CNAO (all relevant in the bump-linux-bridge.sh iirc)